recipes-security/libseccomp/libseccomp_2.4.3.bb

```diff
@@ -0,0 +1,43 @@
+SUMMARY = "interface to seccomp filtering mechanism"
```

Comment:
Why not include meta-security instead of importing a recipe?

commit messages has a typo "libbseccomp" double b.

Reply:
meta-security, the top level layer, doesn't have many bbappends.  There is a busybox one, but that only matters for ptest.  The linux ones only apply to linux-yocto, so those wouldn't affect use.  So at first glance, it seems like it would be okay to include.

dunfell meta-security has libseccomp, but it is removed in the master branch as it moved to core: https://git.yoctoproject.org/cgit/cgit.cgi/meta-security/commit/?id=baca6133f9c7f77a12dc137fe5b90723fbb4c15b